run

```diff
@@ -9,6 +9,7 @@ if [ "$RSA_PRIVATE_KEY" == "" ]; then
 fi
 echo PWD: $PWD
 echo HOME: $HOME
+docker pull rsyslog/docker-alpine-abuild:3.7 # be on the save side...
```

Comment:
Thoughts:

- Update the FROM line in your [Dockerfile](https://github.com/rsyslog/rsyslog-docker/blob/b6136971dc65864a69aa09de8c1bdb10a9914950/appliance/alpine/Dockerfile#L1) to reflect the latest stable upstream image:

```diff
-FROM	alpine:3.7
+FROM	alpine:latest
```

Whenever you build your image the Dockerfile would use the latest upstream stable image.

- Either explicitly tag (or allow the default to apply) your new docker image with 'latest'.

Now when you call the `docker run` command, you could either explicitly specify that tag or allow docker to default to using that tag when fetching the image.

Reply:
We need to be careful here. When 3.8 comes out, we still must support the 3.7 packages, and they **can be different**. For that reason, we need to nail the build to specific versions. The idea is to support all version also currently supported by Alpine. Right now that would be 3.6 and 3.7, but as 3.8 will come out soon, I am not digging into 3.6.

Reply:
some comment applies to the actual packaging content, not just the helper container

Reply:
Gotcha. Carry on then, ignore my chattering. ;)